vuepress/docs/next/docs/getting-started/README.md

```diff
@@ -173,185 +171,135 @@ sudo kubectl create namespace entando
+:::
 
-3. Deploy your config map
+::: tip Entando Operator
+The behavior of the Entando Operator is configurable using an optional ConfigMap named `entando-operator-config`. For example, on a slower network, you may want to increase the download timeouts. Refer to the [Entando Operator](../../tutorials/devops/entando-operator.md) for more information.
+:::
+### Deploy Entando
 
-``` bash
-sudo kubectl apply -f sample-configmaps/entando-operator-config.yaml  -n entando
-```
 
-4. Run Helm and deploy your Entando application
+Deploy Entando by applying `entando-app.yaml` to your namespace.
 
-``` bash
-sudo helm template quickstart ./ | sudo kubectl apply -n entando -f -
+```sh
+sudo kubectl apply -f entando-app.yaml -n entando
 ```
 
 ---
 
-5. Use the `get pods --watch` command to observe Entando starting up
+Use the `get pods --watch` command to observe Entando starting up.
 
-``` bash
-sudo kubectl -n entando get pods --watch
+```sh
+sudo kubectl get pods -n entando --watch
 ```
 
 <details><summary>What does a successful startup look like?</summary>
 
-- First, you'll see the Entando operator: `ContainerCreating` > `Running`
-- Next, the Entando composite app deployer: `Pending` > `ContainerCreating` > `Running`
-- Then, Keycloak: `kc-deployer` > `kc-db-deployment`
+- The `entando-operator` and `entando-k8s-service` are started when you apply the namespace resources above, so they should already be in the `Running` status by the time you reach this point 
+- When you apply the `entando-app.yaml`, the `quickstart-deployer` will start and kick off a series of deployments, beginning with the Keycloak deployment via the `default-sso-in-namespace-deployer`
 
 **Jobs / Deployments**
-- Jobs, like `kc-db-preparation-job`, run once and are `Completed`: `0/1`
-- Database deployments, like `kc-db-deployment`, should end up as `Running`: `1/1`
-- The Keycloak server deployment, `kc-server-deployment`, should end up as `Running`: `1/1`
-- The deployment is done when the `quickstart-composite-app-deployer` pod has a status of `Completed`  
+- Some Pods, like `quickstart-deployer`, run to completion and then shutdown. They should wind up in `Completed`: `0/1`
+- Other deployments, like `quickstart-ab-deployment` or `quickstart-deployment`, should end up as `Running`: `1/1`
+- The deployment is done when the `quickstart-deployer` pod has a status of `Completed`  
 
 **Lifecycle Events**
 - Each line represents an event: `Pending`, `ContainerCreating`, `Running` or `Completed`
-- Restarts should ideally be `0`; otherwise, there was a problem with your cluster, and Kubernetes is trying to self-heal
+- Restarts should ideally be `0`; otherwise, there may be a resource problem (slow network, not enough CPU or memory, etc.) with your cluster, and Kubernetes is trying to self-heal
```

Comment:
the verb tenses are appropriate because of "otherwise." rephrasing it would be something like, "If restarts are not 0, there may be a resource problem, so Kubernetes is trying to self-heal." the non-zero restart status is in the present and Kubernetes is presently/actively self-healing